build/psgallery-release.yml

```diff
@@ -92,8 +93,26 @@ stages:
             parameters:
               repositoryName: '$(Repository)'
               releaseNotes: |
-                Install new version via [NuGet](https://www.nuget.org/packages/$(Project)/$(Build.BuildNumber))
+                Install new version via [PowerShell Gallery](https://www.powershellgallery.com/packages?q=$(Project)
                 ```shell
-                PM > Install-Package $(Project) --Version $(Build.BuildNumber)
+                PS > Install-Module -Name $(Project).* --Version $(Build.BuildNumber)
```

Comment:
Let's not use a `.*` but a concrete example that they can run maybe? For example `Arcus.Scripting.ARM` to align with our README.

Reply:
This makes me wonder, maybe another opportunity for a `Arcus.Scripting` metapackage @mbraekman ?

Reply:
Could be usefull, although most of the times people will only be interested in a specific module, I suppose.
Let's add a task for this to be implemented in one of the future versions.

Reply:
That's a good point!

Reply:
#66 (and similar #67) has been created for this.